packages/editor/src/components/post-saved-state/index.js

```diff
@@ -158,7 +158,7 @@ export default function PostSavedState( {
 			 * of the button that we want to avoid. By setting `showTooltip`,
 			 * the tooltip is always rendered even when there's no keyboard shortcut.
 			 */
-			showTooltip
+			// showTooltip
```

Comment:
I will undo these changes before merging, but for now this helps with testing those changes

Reply:
Yes this is necessary for testing to reproduce the removal of the Button from the DOM, see https://github.com/WordPress/gutenberg/discussions/44735#discussioncomment-7639716

Reply:
Sounds good, let's make sure to not miss removing it before merging.